soccer/gameplay/tactics/defense.py

```diff
@@ -71,6 +113,21 @@ def execute_running(self):
             # raise RuntimeError("Defense tactic requires a goalie id to be set")
 
             # TODO: move a lot of this code into modules in the evaluation folder
+
+            #main.system_state().draw_circle(robocup.Point(0, 0), constants.Field.ArcRadius * 2,constants.Colors.Red, "Clear Ball")
+
+    def execute_clearing(self):
```

Comment:
Good call, I hadn't thought about the goalie.  There is an obstacle created in the GameplayModule that keeps all non-goalie robots out of the goal zone.  This will keep the defenders from actually being able to go into the goal zone, but they'll sort of waffle around on the edge of the goal zone unproductively.  It would be best if they leave the ball to the goalie if it's in the goal zone so that they can focus on defending.